### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by nevin75**

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909&bid=16086904)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909&bid=16086904)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909&bid=16086904)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909&bid=16086904)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        public static void ReportBodyMassIndex(string firstName, double weight, double height)
+        {
+            Console.WriteLine();
+            Console.WriteLine(firstName + "s BMI is " + (CalculateBodyMassIndex(weight, height)) + ".");
+        }
+
+        public static double CalculateBodyMassIndex(double weight, double height)
+        {
+            return weight / height / height);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909&bid=16086904)

File: Src/BootCamp.Chapter/Checks.cs [hunk 2]
```diff
@@ -16,26 +16,26 @@ public static class Checks
...
-            // To do: call your implementation. 
+            Lesson3.RequestFloat(message);
             return 0;
         }
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
+            Lesson3.CalculateBodyMassIndex(weight, height);
             return 0;
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Checks.cs]: You want to return the Lesson3.X(Y) stuff and not those default values.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846909&bid=16086904)

File: Src/BootCamp.Chapter/Checks.cs (hunk 2 above)
Comment (nevin75) [reply Src/BootCamp.Chapter/Checks.cs]: Thanks. I think I've gotten it now.

**Pull request closed without merging**